hcl2template/types.build.provisioners.go

```diff
@@ -21,17 +25,23 @@ func (p *ProvisionerBlock) String() string {
 
 func (p *Parser) decodeProvisioner(block *hcl.Block) (*ProvisionerBlock, hcl.Diagnostics) {
 	var b struct {
-		Name string   `hcl:"name,optional"`
-		Rest hcl.Body `hcl:",remain"`
+		Name        string        `hcl:"name,optional"`
+		PauseBefore time.Duration `hcl:"pause_before,optional"`
```

Comment:
do durations work out of the box just like this ? Please add a test here so that we can see the behaviours this has.

Reply:
I am doing this now

Reply:
btw it doesn't work but I fix it already! just trying to write a test